tests/e2e/test_rosacli_cluster.go

```diff
@@ -1034,3 +1034,58 @@ var _ = Describe("Create cluster with availability zones testing",
 				Expect(common.ReplaceCommaSpaceWithComma(mp.AvalaiblityZones)).To(Equal(availabilityZones))
 			})
 	})
+
+var _ = Describe("Create cluster with fips testing",
+	func() {
+		defer GinkgoRecover()
+		var (
+			clusterID      string
+			rosaClient     *rosacli.Client
+			clusterService rosacli.ClusterService
+		)
+
+		BeforeEach(func() {
+			By("Get the cluster")
+			var clusterDetail *profilehandler.ClusterDetail
+			var err error
+			clusterDetail, err = profilehandler.ParserClusterDetail()
+			Expect(err).ToNot(HaveOccurred())
+			clusterID = clusterDetail.ClusterID
+			Expect(clusterID).ToNot(Equal(""), "ClusterID is required. Please export CLUSTER_ID")
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+		})
+
+		AfterEach(func() {
+			By("Clean remaining resources")
+			rosaClient.CleanResources(clusterID)
+
+		})
+
+		It("Rosa cluster with fips enabled can be created successfully - [id:46312]",
+			labels.Critical, labels.Runtime.Day1Post,
+			func() {
+				clusterName := "ocp-46312"
+				profile := profilehandler.LoadProfileYamlFileByENV()
+
+				if !profile.ClusterConfig.FIPS {
+					SkipTestOnFeature("create rosa cluster with fips")
+				}
+
+				By("Check if etcd_encryption is enabled")
+				output, err := clusterService.DescribeCluster(clusterID, "-o", "yaml")
+				des, err := clusterService.ReflectClusterDescription(output)
+
+				Expect(err).ToNot(HaveOccurred())
+				Expect(des.EtcdEncryption).To(Equal("true"))
+
+				By("Create cluster with fips and '--etcd-encryption=false'")
```

Comment:
This is another day1-negative case

Reply:
split to another day1-negtive case(ocp-74436)